src/js/tech/html5.js

```diff
@@ -573,6 +577,33 @@ class Html5 extends Tech {
   }
 
   /**
+   * Proxy iOS `webkitbeginfullscreen` and `webkitendfullscreen` into
+   * `fullscreenchange` event
+   *
+   * @private
+   * @method proxyWebkitFullscreen_
+   */
+  proxyWebkitFullscreen_() {
+    if ('webkitDisplayingFullscreen' in this.el_) {
+      const endFn = function() {
+        this.trigger('fullscreenchange', { isFullscreen: false });
+      };
+
+      const beginFn = function() {
+        this.one('webkitendfullscreen', endFn);
+
+        this.trigger('fullscreenchange', { isFullscreen: true });
+      };
+
+      this.on('webkitbeginfullscreen', beginFn);
+      this.on('dispose', () => {
+        this.off('webkitbeginfullscreen', beginFn);
+        this.off('webkitendfullscreen', endFn);
```

Comment:
will this trigger an error if we are not in full screen? aka `this.one('webkitendfullscreen', endFn);` is already triggered and gone or was never added in the first place(in the case of never going to full screen)